src/_backend_agg.h

```diff
@@ -1178,12 +1171,12 @@ inline void RendererAgg::draw_quad_mesh(GCAgg &gc,
                                   offsets,
                                   offset_trans,
                                   facecolors,
-                                  *edgecolors_ptr,
+                                  edgecolors,
                                   linewidths,
                                   linestyles,
                                   antialiaseds,
                                   OFFSET_POSITION_FIGURE,
-                                  false,
+                                  true, // snap
```

Comment:
This actually only enables checking whether the snap flag is set, rather than enabling snapping (I think); perhaps the comment could be slightly clearer? e.g. just replace `snap` by `check_snap`.

Reply:
Yes, that is correct. Updated with the suggestion.